sdk/compute/arm-compute/README.md

```diff
@@ -1,89 +1,91 @@
 ## Azure ComputeManagementClient SDK for JavaScript
 
-This package contains an isomorphic SDK for ComputeManagementClient.
+This package contains an isomorphic SDK (runs both in node.js and in browsers) for ComputeManagementClient.
 
 ### Currently supported environments
 
-- Node.js version 6.x.x or higher
+- Node.js version 8.x.x or higher
 - Browser JavaScript
 
-### How to Install
+### How to install
 
+To use this SDK in your project, you will need to install two packages.
+- `@azure/arm-compute` that contains the client.
+- `@azure/identity` that contains different credentials for you to authenticate the client using Azure Active Directory.
+
+Install both packages using the below commands.
 ```bash
 npm install @azure/arm-compute
+npm install @azure/identity
 ```
+Please note that while the credentials from the older [`@azure/ms-rest-nodeauth`](https://www.npmjs.com/package/@azure/ms-rest-nodeauth) and [`@azure/ms-rest-browserauth`](https://www.npmjs.com/package/@azure/ms-rest-browserauth) packages are still supported, these packages are in maintenance mode receiving critical bug fixes, but no new features.
+We strongly encourage you to use the credentials from `@azure/identity` where the latest versions of Azure Active Directory and MSAL APIs are used and more authentication options are provided.
 
 ### How to use
 
-#### nodejs - client creation and list operations as an example written in TypeScript.
-
-##### Install @azure/ms-rest-nodeauth
-
-- Please install minimum version of `"@azure/ms-rest-nodeauth": "^3.0.0"`.
-```bash
-npm install @azure/ms-rest-nodeauth@"^3.0.0"
-```
+There are multiple credentials available in the `@azure/identity` package to suit your different authentication needs.
+Read about them in detail in [readme for @azure/identity package](https://www.npmjs.com/package/@azure/identity).
+To get started you can use the [DefaultAzureCredential](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/identity/identity/README.md#defaultazurecredential) which tries different credentials internally until one of them succeeds.
+Most of the credentials would require you to [create an Azure App Registration](https://docs.microsoft.com/en-us/azure/active-directory/develop/app-objects-and-service-principals#application-registration) first.
```

Comment:
We may need to mention that most scenarios will require that the App Registration has the correct permissions. For example, the "Azure Service Management" permission.